pkg/mesh-discovery/translation/mesh/detector/istio/istio_mesh_detector_test.go

```diff
@@ -277,6 +277,7 @@ var _ = Describe("IstioMeshDetector", func() {
 						TrustDomain:          trustDomain,
 						IstiodServiceAccount: serviceAccountName,
 						IngressGateways: []*discoveryv1.MeshSpec_Istio_IngressGatewayInfo{{
+							Namespace:      "namespace",
 							WorkloadLabels: workloadLabels,
```

Comment:
Name is now part of the discovered mesh's IngressGatewayInfo
```suggestion
							Namespace:      "namespace",
							Name:           "ingress-svc",
							WorkloadLabels: workloadLabels,
```